### PR TITLE
[DO NOT MERGE] Spike 3DS

### PR DESCRIPTION
**Pull request by sshropshire**
### Summary of changes

 - Spike to initiate a 3DS Verification Flow
 - This requires an `application_context` when creating an Order via _ppcp_merchant_server_
 - For now, we're forwarding these parameters to the merchant server
 - In the future, we should find a way to set `return_url` and `cancel_url` when calling `confirm-payment-source` to avoid requiring a merchant to provide a server-side integration in order to perform 3DS Verification

 ### Checklist

 ~- [ ] Added a changelog entry~

### CODE REVIEW

File: Card/src/main/java/com/paypal/android/card/CardAPIRequestFactory.kt [hunk 1]
```diff
@@ -35,4 +35,43 @@ internal class CardAPIRequestFactory {
         val path = "v2/checkout/orders/$orderID/confirm-payment-source"
         return APIRequest(path, HttpMethod.POST, body)
     }
+
+    fun createAuthorizeWith3DSVerificationRequest(orderID: String, card: Card): APIRequest {
+        val cardNumber = card.number.replace("\\s".toRegex(), "")
```
Comment (crookedneighbor) [inline Card/src/main/java/com/paypal/android/card/CardAPIRequestFactory.kt:40]: Should this be changed to remove any non-number? And should this logic for cleaning up the card number be moved to a function that can be used anywhere it needs to be run? (I assume non-3ds card flows will also need to do it)

**Comment by crookedneighbor** [comment]
Would love to see what the API looks like for the merchant. Do we have any mock up docs for what a merchant would do to integrate with 3ds?

File: Card/src/main/java/com/paypal/android/card/CardAPIRequestFactory.kt (hunk 1 above)
Comment (sshropshire) [reply Card/src/main/java/com/paypal/android/card/CardAPIRequestFactory.kt:40]: We may be missing a spec for this. For now all it does is strip out spaces, but we should have local validation in place to provide instant feedback.

**Comment by sshropshire** [comment]
> Would love to see what the API looks like for the merchant. Do we have any mock up docs for what a merchant would do to integrate with 3ds?

Not at the moment. Currently, 3DS requires `return_url` and `cancel_url` to be forwarded to the create order POST, which would require us to provide a server side example. The assumption is that we'll be able to provide these values to `confirm-payment-source` in the near future and it'll just work.

**Comment by sshropshire** [comment]
Closing in favor of #62.

**Pull request closed without merging**